user/user.go

```diff
@@ -110,16 +114,36 @@ func (env *env) createUserHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	user, err := env.dao.CreateUser(dao.CreateUserInput{
+	input := dao.CreateUserInput{
 		ID:   auth.ID,
 		Name: req.Name,
-	})
+	}
+
+	for _, hook := range env.hook.beforeCreateHooks{
```

Comment:
Is this really the gofmt-approved spacing before the open-brace?

Reply:
Good catch, IntelliJ doesn't have gofmt turn on for save </3